central/graphql/resolvers/images_test.go

```diff
@@ -217,22 +265,57 @@ func (s *ImageResolversTestSuite) TestDeployments() {
 
 				// Test ImageCVECountBySeverity for each deployment resolver.
 				expectedCVESevCount := compileExpectedCountBySeverity([]*storage.Image{expectedImages[imageID]}, tc.vulnFilter)
-				actualCVECnt, err := image.ImageCVECountBySeverity(ctx, RawQuery{Query: paginatedQ.Query})
+				actualCVECnt, err := image.ImageCVECountBySeverity(testCtx, RawQuery{Query: paginatedQ.Query})
 				assert.NoError(t, err)
```

Comment:
Use `require` instead of `assert` to prevent access of nil variables. As well as lines below.